tools/topology/topology2/sof-hda-generic.conf

```diff
@@ -48,6 +54,18 @@ Define {
 	DMIC0_DAI_GAIN 'eqiir.12.1'
 	DMIC0_DAI_EQIIR "highpass_40hz_20db"
 	DMIC0_PCM_CAPS 'Gain Capture 11'
+	DMIC1_PCM_CAPS	'DMIC1 WOV Capture'
+	DMIC1_HOST_PIPELINE_ID		16
+	DMIC1_DAI_PIPELINE_ID		17
+	WOV_PIPELINE_ID			18
+	DMIC1_HOST_PIPELINE_SINK	'copier.host.16.1'
+	DMIC_WOV_DAI_PIPELINE_SRC	'copier.DMIC.17.1'
+	DMIC_WOV_DAI_PIPELINE_KPB	'kpb.17.1'
+	WOV_PIPELINE_SINK		'micsel.18.1'
```

Comment:
The dmic-default.conf defines this:
`WOV_PIPELINE_SINK               'micsel.20.1'`
and most of these are also found in there, so we change them here? Why not in there?

Reply:
dmic-default.conf defines the default values used commonly for dmic, in most cases, if there is exception, we should do the change in the top-level topology.